llarp/handlers/tun.cpp

```diff
@@ -315,19 +237,9 @@ namespace llarp
     void
     TunEndpoint::Flush()
     {
-      static const auto func = [](auto self) {
-        self->FlushSend();
```

Comment:
capture `this` ...?

Reply:
looks like it's capturing a shared_from_this using std::bind below